# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../ulogd2/0002-include-linux_types.h.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

include <linux/types.h> to get __be{16,32} types

error log
------------------------------------------------------------------------------
ulogd_raw2packet_BASE.c:591:2: error: unknown type name '__be16'
ulogd_raw2packet_BASE.c:592:2: error: unknown type name '__be16'
ulogd_raw2packet_BASE.c:593:2: error: unknown type name '__be32'
ulogd_raw2packet_BASE.c:594:2: error: unknown type name '__be32'
------------------------------------------------------------------------------

--- a/filter/raw2packet/ulogd_raw2packet_BASE.c	2013-01-02 02:00:55.022574242 +0100
+++ b/filter/raw2packet/ulogd_raw2packet_BASE.c	2013-01-02 02:01:23.535273074 +0100
@@ -48,6 +48,7 @@
 #include <ulogd/ipfix_protocol.h>
 #include <netinet/if_ether.h>
 #include <string.h>
+#include <linux/types.h>
 
 enum input_keys {
 	INKEY_RAW_PCKT,
